app/api/packets/helpers.ts

```diff
@@ -1,8 +1,11 @@
 import { Packet, PacketStates } from 'utils/types/packet';
-import logger from 'utils/logger';
 import { packet } from '../utils/cosmos/polyibc';
+import logger from 'utils/logger';
 
 function stringToState(state: string) {
+  if (!state) {
+    return PacketStates.SENT;
+  }
```

Comment:
Why do you need this if `default` in switch statement does the same?

Reply:
Good catch, already fixed this somewhere else